docs/sources/panels-visualizations/visualizations/table/index.md

```diff
@@ -31,6 +31,10 @@ Tables are very flexible, supporting multiple modes for time series and for tabl
 
 {{< figure src="/static/img/docs/tables/table_visualization.png" max-width="1200px" lightbox="true" caption="Table visualization" >}}
 
+The following video will give you a visual walkthrough over the configurations and settings you can set in a table panel. If you get stuck not knowing where is a setting in the documentation or want to see a configuration in action, check it out in the video below.
```

Comment:
I took out the "if you get stuck in the documentation" part because I don't think we want to suggest that people are going to get stuck in the docs. Otherwise, I've just made some style fixes.

```suggestion
The following video provides a visual walkthrough of the configurations and settings you can set in a table visualization. If you want to see a configuration in action, check it out in the video:
```